### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by jonashendrickx**
This will assign our team to the 'Reviewers' section automatically, similarly, workflow changes would be automatically reviewed by CloudOps.

### CODE REVIEW

**Review by abergs — CHANGES_REQUESTED** [review]
Please add description and reasoning for suggesting this change.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# For example, default owners should always be the first group.
+# Sort lines alphabetically within these groups to avoid accidentally adding duplicates.
+#
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# Default file owners
+* @bitwarden/team-passwordless-dev
+
+# DevOps for Actions and other workflow changes
+.github/workflows @bitwarden/dept-devops
```
Comment (abergs) [inline .github/CODEOWNERS:11]: Comment this out for now

**Review by abergs — CHANGES_REQUESTED** [review]

**Review by abergs — COMMENTED** [review]
I think this looks fair now, but as I don't know anything about codeowners, maybe @Tyrrrz could confirm and approve

**Review by Tyrrrz — APPROVED** [review]
Looks fine in the final state

**Comment by jrmccannon** [comment]
It puts all members of our team into the list of people who have to approve a PR for it to go in, but then the workflow addition allows devops to make changes to the workflow pipelines without bothering us. It is inline with the rest of the organization.

**Review by jrmccannon — APPROVED** [review]

**Comment by Tyrrrz** [comment]
> but then the workflow addition allows devops to make changes to the workflow pipelines without bothering us.

That's not really how it works, they can already do that now. Instead, it will make it so all changes to the workflow will tag the devops team.

**Pull request merged**